recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaFilters.java

```diff
@@ -79,7 +79,7 @@ public Object apply(Object value, Object... params) {
                             case FLOAT:
                                 return super.asString(value);
                             case STRING:
-                                return "\"" + super.asString(value) + "\"";
+                                return "\"" + super.asString(value).replace("\"", "\\\"") + "\"";
```

Comment:
Hm... what about things like newlines in the strings?

Maybe using backticks will solve all this? Assuming node supports them...

Reply:
Hm, I think I'll leave that for now..

Reply:
I'd say put this `.replace(...)` logic in a helper that both places can call. Maybe let that helper do the `"\"" +` logic as well. Then in the future you can just modify that helper to switch to backticks or to single quotes etc.  This can all be done later though, it's only in 2 places atm.

Reply:
I wonder if we should do that given that it could be language specific? Each language we support may want to escape differently.

Reply:
Yeah this helper would be JS specific

Reply:
Currently we have one for Java and one for TypeScript/JS. I think it makes sense to keep those separate.

Reply:
Oh... I get it! I was thinking they were both for TS.  They do both have a bug with things like newlines in the string though, but that could be fixed on it's own later.

Reply:
Yup, that's what I was thinking. Too much other core stuff to deal with other than newlines in strings for now :P